scanner/scanner.go

```diff
@@ -11,6 +11,23 @@ import (
 	"runtime/debug"
 )
 
+//sql.Rows implement this interface
+type Rows interface {
```

Comment:
It's good, but Is this interface too large? I mean, the purpose of using an interface is to make replacement easier, and scanner doesn't use methods like `Err` and `NextResultSet` . When people wants to create a Rows type they have to implement those unused methods.
So I suggest remove `ColumnTypes ` `Err` and `NextResultSet ` from the definition.

Reply:
you are right.